hack/pre-release.sh

```diff
@@ -148,6 +154,124 @@ update_OCP_version() {
   fi 
 }
 
+# This function handles updating the versions in GitHub
+# takes 3 arguments
+#
+# 1st arg: base branch to operate on 
+# 2nd arg: stream to update. Can be x, y or z to represent the x y or z stream.#
+# 3rd arg: current WMCO version
+github_update() {
+  if [ "$#" -ne 3 ]; then
+    echo incorrect parameter count for github_update $#
+    return 1
+  fi
+
+  base_branch=$1
+  stream=$2
+  wmco_version=$3
+
+  # check if the git branch is dirty, and exit if it is
+  check_git_branch
```

Comment:
Will this exit here too or just exit common.sh? (also asking for the two other places this is called)